Library/Formula/mitie.rb

```diff
@@ -14,27 +14,34 @@ class Mitie < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
+  option "with-models", "Install v0.2 models (~415MB)"
```

Comment:
I'm not 100 percent sure! But I assumed something of that size shouldn't be downloaded by default.